Sources/MapboxDirections/DirectionsOptions.swift

```diff
@@ -99,6 +99,7 @@ public struct DirectionsPriority: Hashable, RawRepresentable {
     /**
      The priority level with which a route neither avoids nor prefers a particular type of roadway or pathway.
      */
+    // should we remove this default (or change its naming) if we don't set it as such?
```

Comment:
Yes, `medium` (as in “medium priority”) would be better, since apparently the API reserves the right to use a different default at some point in the future, even if that would be somewhat nonsensical for a relative value.